avcheck.py

```diff
@@ -0,0 +1,606 @@
+                        "string",
+                        max_length=proc.ImageFileName.vol.count,
+                        errors="replace"
+                    )
+                    
+                    for dll in proc.load_order_modules():
+                        try:
+                            dll_name = dll.BaseDllName.get_string().lower()
+                            dll_path = dll.FullDllName.get_string()
+                            
+                            if any(pattern in dll_name for pattern in dll_patterns):
+                                dll_key = f"{dll_name}|{dll_path}"
+                                if dll_key not in seen_dlls:
+                                    seen_dlls.add(dll_key)
+                                    av_dlls.append({
+                                        'name': dll_name,
+                                        'path': dll_path,
+                                        'process': proc_name,
+                                        'pid': proc.UniqueProcessId,
+                                        'base': format_hints.Hex(dll.DllBase)
+                                    })
+                                    
+                        except Exception:
+                            continue
+                            
+                except Exception:
+                    continue
+                    
+        except Exception as e:
+            vollog.debug(f"Error scanning DLLs: {e}")
+        
+        return av_dlls
+    
+    def _detect_av_files(self) -> List[Dict]:
+        """Detect antivirus-related files including detection logs and quarantine."""
+        av_files = []
+        
+        try:
+            # Use filescan to find AV-related files
+            for file_obj in filescan.FileScan.scan_files(
+                self.context,
+                self.config["kernel"]
+            ):
+                try:
+                    file_name = file_obj.FileName.String
+                    file_name_lower = file_name.lower()
+                    
+                    # Check if file matches AV patterns (stricter filtering)
+                    is_detection_file = any(
+                        pattern.lower() in file_name_lower 
+                        for pattern in self.DETECTION_PATTERNS
+                    )
+                    
+                    # Check for AV vendor directories (stricter)
+                    av_directories = [
+                        'windows defender', 'kaspersky', 'mcafee', 'norton', 'symantec', 
+                        'bitdefender', 'eset', 'malwarebytes', 'sophos', 'trend micro',
+                        'avast', 'avg', 'avira', 'comodo', 'fsecure', 'panda', 'gdata',
+                        'webroot', 'quarantine', 'virus', 'threat', 'infected'
+                    ]
+                    
+                    is_av_directory = any(av_dir in file_name_lower for av_dir in av_directories)

```

Comment:
This is very permissive.  Any file that contains any of these values (such as the word virus to threat, in the filename).  Does this return useful results?